src/main/java/jenkins/plugins/slack/CommitInfoChoice.java

```diff
@@ -0,0 +1,61 @@
+package jenkins.plugins.slack;
+
+public enum CommitInfoChoice {
+    NONE("nothing about commits") {
```

Comment:
Frankly speaking, I know nothing about localization in Jenkins.
According to [Internationalization page](https://wiki.jenkins-ci.org/display/JENKINS/Internationalization), there must be a *.properties file in the source tree, so I doubt this plugin currently has support for localization.

While it's possible to add this support, I think the change deserves a separate pull request.

I can convert them to String constants though if it helps in the future to add l10n support.